apiserver/resources/server_addpending_test.go

```diff
@@ -293,24 +292,24 @@ func (s *AddPendingResourcesSuite) TestWithURLUpload(c *gc.C) {
 //func (s *AddPendingResourcesSuite) TestUnknownResource(c *gc.C) {
```

Comment:
Not sure why this was originally commented out. 
We should either skip with a bug and a todo, or delete.